_develop/collections.md

```diff
@@ -0,0 +1,217 @@
+   "website": "",
+   "telephone": "",
+   "identifier": "sally jeffries"
+}
+```
+
+A collection schema is neither validated or enforced. The goal is to incentivize collection use rather that enforce use. Developers have the ability to roll-back in case apps make undesirable changes that break compatibility with collections. To support this rollback in the pre-release, collections data store is conceptually an event log. Every data write an app makes is stored as a separate file. By placing data in files it ensures that data is never lost and developers can return files back to any previous state. 
+
+<div class="uk-card uk-card-default uk-card-body">
+<h5 class="uk-card-title">The Future of Collections Envisioned</h5>
+<p>Blockstack believes that collections feature of the Blockstack platform can reduce the damage that faulty and malicious apps can cause to user’s data. For end=users, Blockstack seeks to enable:</p>
+<ul class="uk-list uk-list-bullet uk-text-small">
+  <li>True data portability across applications for each distributed ID</li>
+  <li>Management of application access and permissions to collection data.</li>
+</ul>
+<p>For developers, collections can incentivize user adoption by reducing user friction.  Users can easily try new apps and move to them without the overhead or barrier of re-entering data. You are <a href="https://forum.blockstack.org/t/feedback-wanted-collections-design/7752" target="_blank">welcome to review and comment</a> on the current design document.</p>
+</div>
+
+## Try the Contact Manager demo app
+
+Before adding collections to your DApp, you can try it for yourself using the Contact Manager demo application. Blockstack Contacts is a simple contacts manager that allows users to add and manage their contacts. The data stored by this app can be used in another app that receives the contacts collection permissions.
+
+The tutorial relies on the `npm` dependency manager. Before you begin, verify
+you have installed `npm` using the `which` command to verify.
+
+```bash
+$ which npm
+/usr/local/bin/npm
+```
+
+If you have `npm` installed, do the following to run the Contact Manager demo app:
+
+1. If you have a local Blockstack installed, <a href="{{"/browser/local_browser.html" | prepend:site.baseurl}}" >uninstall it</a>.
+
+2. Download and install the <a href="https://github.com/blockstack/blockstack-browser/releases/tag/collections-alpha.1" target="_blank">Collections Alpha Build</a> of the Blockstack Browser client for your OS.
+
+3. Launch the alpha build of the local Blockstack Browser client.
+
+   {% include warning.html content="This alpha build interacts with a version of the Gaia Storage hub that supports collections. You cannot use an existing ID to test with, instead, you need to <strong>create a new, test ID using the alpha build</strong>. Additionally, <strong>you must create a custom Gaia hub during the Contact Manager onboarding</strong>. For this reason, wait to create a new ID." %}
```

Comment:
They don't necessarily need to create a custom Gaia hub. They just need to connect to the custom one that we put up for the preview.

Reply:
Ok, I'll clarify.